contracts/SmartWalletImplementation.sol

```diff
@@ -198,7 +242,7 @@ contract SmartWalletImplementation is SmartWalletStorage, ISmartWalletImplementa
             diff = params.destAmount - destAmount;
         }
 
-        // Telerate a 5% difference
+        // Tolerate a 5% difference
         require(diff < params.destAmount / 20, "getExpectedIn_noResult");
```

Comment:
This is not related to price impact though, it was generic calculation of `srcAmount` given `destAmount` using binary search. Because we cannot run for infinite number of loops, the calculation just return the best "guessed" `srcAmount`, this check just to prevent unreasonable `srcAmount`